admin/class-metabox.php

```diff
@@ -182,69 +182,21 @@ public function add_meta_box() {
 	}
 
 	/**
-	 * Pass some variables to js for the edit / post page overview, snippet preview, etc.
+	 * Pass variables to js for use with the post-scraper
 	 *
-	 * @return  array
+	 * @return array
 	 */
-	public function localize_script() {
-		$post = $this->get_metabox_post();
-
-		if ( ( ! is_object( $post ) || ! isset( $post->post_type ) ) || $this->is_metabox_hidden( $post->post_type ) === true ) {
-			return array();
-		}
-
-		$options = get_option( 'wpseo_titles' );
-
-		$date = '';
-		if ( isset( $options[ 'showdate-' . $post->post_type ] ) && $options[ 'showdate-' . $post->post_type ] === true ) {
-			$date = $this->get_post_date( $post );
-
-			self::$meta_length        = ( self::$meta_length - ( strlen( $date ) + 5 ) );
-			self::$meta_length_reason = __( ' (because of date display)', 'wordpress-seo' );
-		}
-
-		self::$meta_length_reason = apply_filters( 'wpseo_metadesc_length_reason', self::$meta_length_reason, $post );
```

Comment:
I have documented the filter here: https://github.com/Yoast/wordpress-seo/issues/2795. Let's discuss deprecation on monday.